addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -2331,6 +2346,50 @@ export class OdooEditor extends EventTarget {
                 }
             }
         } else if (node.nodeType !== Node.TEXT_NODE) {
+            if (node.nodeName === 'TD' && node.hasAttribute('style')) {
+                // TD tags do not support style in v15, move style to children or new SPAN.
+                const spans = node.childNodes.length === 0 || [...node.childNodes].some(n => !['SPAN', 'FONT'].includes(n.nodeName))
+                    ? [this.document.createElement('SPAN')]
+                    : [...node.childNodes];
+                node.style.backgroundColor = null; // background-color: on TD (cell) != on span (text).
+                spans.forEach(span => {
+                    node.getAttribute('style').split(';').forEach(styleText => {
+                        // Give parent's style to span child, unless it already exists.
+                        const [styleName, styleValue] = styleText.split(':').map(x => x.trim());
+                        if (!span.style[styleName]) {
```

Comment:
I don't think that's how it works because IIRC `'hyphened-style: whatever;'` becomes `style.hyphenedStyle = 'whatever'`.

Reply:
Oh ok, looks like the `style` property accepts both format somehow. Didn't know that. ^^'